firmware/Core/Inc/mpi/mpi_types.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * mpi_types.h
+ *
+ * Created on: Jun 14, 2024
+ * Author: Vaibhav Kapoor
+ */
+
+#ifndef __INCLUDE_GUARD__MPI_TYPES_H
+#define __INCLUDE_GUARD__MPI_TYPES_H
+#include <stdint.h>
+
+/**
+ * @enum Mpi_Mode
+ * @brief Enumerates the different modes under which the satellite can operate the MPI
+ * 
+ */
+enum Mpi_Mode {
+    COMMAND_MODE,                           // MPI is in command mode
+    SCIENCE_DATA_MODE,                      // MPI is science data is being collected mode
+    NOT_LISTENING_TO_MPI_MODE               // MPI is recording science data but it is not being collected mode
+};
+
+extern enum Mpi_Mode current_mpi_mode;      // Current mode under which the MPI is being operated
```

Comment:
This extern should not be in the `mpi_types.h` file. Please move to `mpi_command_handling.h`